packages/core/src/metas/initialize.ts

```diff
@@ -2,19 +2,36 @@ import type { Config } from '../config';
 import type { InternalLogger } from '../internalLogger';
 import { isFunction } from '../utils';
 import { VERSION } from '../version';
-import type { Meta, Metas, MetaItem } from './types';
+import type { Meta, Metas, MetaItem, MetasListener } from './types';
 
 export function initializeMetas(internalLogger: InternalLogger, config: Config): Metas {
   let items: MetaItem[] = [];
+  let listeners: MetasListener[] = [];
+
+  const _getValue = () => {
```

Comment:
Underscore variables are usually the ones marked as "not used" rather than internal. Let's keep it camelcase.